pkg/consensus/payload_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/nspcc-dev/dbft/payload"
 	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
 	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/internal/testutil"
```

Comment:
Maybe it's better to call it `testio` or `testserdes`, `testutil` is too generic and it can provoke people to add more unrelated things there.

Reply:
Done! Initial reason was that we could have more test helpers if needed.